src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -91,6 +93,7 @@ class SymfonyRuntime extends GenericRuntime
      *   dotenv_overload?: ?bool,
      *   dotenv_extra_paths?: ?string[],
      *   worker_loop_max?: int, // Use 0 or a negative integer to never restart the worker. Default: 500
+     *   worker_middlewares?: string
```

Comment:
the option itself should probably support passing an array of strings directly, instead of forcing to implode it with new lines.

Reply:
Now is both possible string and array